### PR TITLE
updating to python 3.9

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

**Comment by ltclm** [comment]
does that mean that all the projects here:
https://github.com/search?q=org%3Ageoadmin%20gatilegrid&type=code
have to be updated to python3.11 (including prewarm.py on geodatasync, etc.)?

If yes i would prefer to go to python3.9 instead which is available on our infra.

File: gatilegrid/__init__.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 from .tilegrids import GeoadminTileGridLV03, GeoadminTileGridLV95, \
     GlobalMercatorTileGrid, GlobalGeodeticTileGrid
-from .grid import Grid
```
Comment (procrastinatio) [inline gatilegrid/__init__.py:3]: Why removing `Grid`?

File: setup.py [hunk 2]
```diff
@@ -1,46 +1,34 @@
...
-      zip_safe=False,
-      long_description=long_description,
-      long_description_content_type='text/markdown',
-      test_suite='nose.collector',
-      install_requires=['future'],
-      python_requires='>2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4',
-      )
+setup(
+    name='gatilegrid',
+    version='0.2.1',
```
Comment (procrastinatio) [inline setup.py]: If you remove `python2`, I would bump the version a bit more

**Review by procrastinatio — CHANGES_REQUESTED** [review]
As the code is published on `pypi`, I would be extremly conservative regarding what version to keep and ditch.

File: gatilegrid/__init__.py (hunk 1 above)
Comment (rebert) [reply gatilegrid/__init__.py:3]: as it is not being used

**Comment by procrastinatio** [comment]
@ltclm Most project use a pinned version of the library, so it should'nt be an issue.

**Review by procrastinatio — APPROVED** [review]
OK, if you want. Do not forget to publish to `PyPi`

**Pull request closed without merging**